vgen/src/main/java/io/gapi/vgen/GeneratorBuilderUtil.java

```diff
@@ -36,12 +38,24 @@ private GeneratorBuilderUtil() {}
     void error(String message, Object... args);
   }
 
-  public static GapicLanguageProvider createLanguageProvider(ConfigProto configProto, final Model model) {
+  /**
+   * Constructs a language provider from a configuration; the configuration cannot contain
+   * parameterized type information, so we must suppress the type safety warning here.
+   */
+  @SuppressWarnings("unchecked")
+  public static GapicLanguageProvider<ProtoElement> createLanguageProvider(
+      ConfigProto configProto,
+      TemplateProto template,
+      final Model model,
+      final InputElementView<ProtoElement> view) {
```

Comment:
I buy that it's generally a good idea to make parameters that don't need to be mutated `final` for readability, but I'm removing the `final` here for consistency with the other non-model parameters.